tests/unit/test_data/test_data.py

```diff
@@ -0,0 +1,80 @@
+from requirements.requirement import Requirement
```

Comment:
Haven't we agreed to have
```
tests
tests_integrations
```

In such approach we don't need `unit` directory. WDYT?

Reply:
Refactored